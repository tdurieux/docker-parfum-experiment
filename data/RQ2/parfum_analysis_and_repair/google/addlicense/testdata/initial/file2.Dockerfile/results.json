{
  "startTime": 1674219688858,
  "endTime": 1674219689632,
  "originalSmells": []
}