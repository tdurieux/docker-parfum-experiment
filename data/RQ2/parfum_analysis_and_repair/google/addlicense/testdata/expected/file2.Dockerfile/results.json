{
  "startTime": 1674215773935,
  "endTime": 1674215774039,
  "originalSmells": []
}